content/200-concepts/100-components/01-prisma-schema/06-relations/425-referential-actions/index.mdx

```diff
@@ -31,7 +31,7 @@ If you do not specify a referential action, Prisma [uses a default](#referential
 
 <Admonition type="alert">
 
-It is extremely important that you check the [upgrade paths for referential actions](/guides/upgrade-guides/upgrading-versions/upgrading-to-prisma-3/referential-actions) section before you continue. The `referentialActions` feature **removes the safety net in Prisma Client that prevents cascading deletes at runtime**. If you use the feature _without upgrading your database_, the [old default action](/guides/upgrade-guides/upgrading-versions/upgrading-to-prisma-3/referential-actions#prisma-2x-default-referential-actions) - `ON DELETE CASCADE` - becomes active. This may result in cascading deletes that you did not expect.
+If you upgrading from a Prisma version before 3.0.1 (or 2.26.0 if you are planning to activate the `referentialActions` preview feature), it is extremely important that you check the [upgrade paths for referential actions](/guides/upgrade-guides/upgrading-versions/upgrading-to-prisma-3/referential-actions) section before you continue. The `referentialActions` feature **removes a previous safety functionality in Prisma Client that prevents cascading deletes at runtime**. If you use the feature _without upgrading your database_, the [old default action](/guides/upgrade-guides/upgrading-versions/upgrading-to-prisma-3/referential-actions#prisma-2x-default-referential-actions) - `ON DELETE CASCADE` - becomes active. This may result in cascading deletes that you did not expect.
```

Comment:
(Beginning of paragraph) This is still not great wording, but I can not figure out a better way to reflect reality.

Reply:
It's near-great wording, though. It's a lot to explain for sure. Consider this version:

If you are upgrading from a Prisma version earlier than 3.0.1 (or 2.26.0 if you are planning to activate the `referentialActions` preview feature), it is extremely important that you review the documentation on [upgrade paths for referential actions](/guides/upgrade-guides/upgrading-versions/upgrading-to-prisma-3/referential-actions) section before you continue. The `referentialActions` feature **removes a previous safety functionality in Prisma Client that prevents cascading deletes at runtime**. If you use the feature _without upgrading your database_, the [old default action](/guides/upgrade-guides/upgrading-versions/upgrading-to-prisma-3/referential-actions#prisma-2x-default-referential-actions) - `ON DELETE CASCADE` - becomes active. This might result in cascading deletes that you did not expect.

Reply:
Added your version as a suggestion to your comment. Definitely works.